Instagram UI added

abhichauhan12 opened the pull request:
Instagram UI added

Commit:
Instagram UI added

anonrootkit commented:
Changes to be done : 
- Open StartingActivity, if back pressed on CreateAccount/Login Activity.
- Open Login Activity, if back pressed after clicking on "sign up" button in Login Activity
- Open CreateAccount Activity, if back pressed after clicking on "Login" button in CreateAccount Activity

Commit:
Changes to be done :

Open StartingActivity, if back pressed on CreateAccount/Login Activity.
Open Login Activity, if back pressed after clicking on "sign up" button in Login Activity
Open CreateAccount Activity, if back pressed after clicking on "Login" button in CreateAccount Activity

anonrootkit commented:
More changes to be done : 
- Add scrollview in both Login/CreateAccount activity layouts

Commit:
Add scrollview in both Login/CreateAccount activity layouts

anonrootkit commented:
Changes to be done :
- Reduce some margin from Login/CreateActivity activity layout

Commit:
Reduce some margin from Login/CreateActivity activity layout

anonrootkit commented:
Changes to be done : 
- Change StartActivity name to WelcomeActivity and its layout name to activity_welcome.

Commit:
Change StartActivity name to WelcomeActivity and its layout name to activity_welcome
